src/cmd/tools/dtest/docker/harness/query_api_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package harness
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/m3db/m3/src/cmd/tools/dtest/docker/harness/resources"
+)
+
+var instantQueryBadRequestTest = []struct {
+	name string
+	q    string
+}{
+	// {"missing query", queryURL("query", "")},
```

Comment:
Why `missing query` is commented out? If it is because of the same `FAILING issue #2`, it should move below the comment (and perhaps the comment itself should be adjusted to account for that).

Reply:
https://github.com/m3db/m3/pull/2873/commits/77f076c2efb862b562c7c349e6247d21a2ba78d3#diff-391a291f1f91098bc731847f8843207e5327fe87a52a4599a45ac0cf4c5732e7R42-R44